Sources/AmazonLocationiOSAuthSDK/Types/ApiTypes/Endpoints/SearchByPositionEndpoint.swift

```diff
@@ -0,0 +1,21 @@
+public struct SearchByPositionEndpoint: AmazonLocationEndpoint {
```

Comment:
Why are we including this in the authentication SDK? It sounds like this should go into one of our sample applications.

Reply:
Removed from the auth sdk and moved to quick start app